udata_front/settings.py

```diff
@@ -11,6 +11,13 @@
 # Default page size for resources on dataset page
 RESOURCES_DEFAULT_PAGE_SIZE = 6
 
+# Default page size for resources on dataset page
+RESOURCES_MIN_COUNT_TO_SHOW_SEARCH = 12
+
+# Autocomplete parameters
+SEARCH_AUTOCOMPLETE_ENABLED = True
```

Comment:
Don't we want to use the values from udata?

Reply:
I forgot the udata PR to move them here : https://github.com/opendatateam/udata/pull/2716

Reply:
We've kept these settings on udata side, we should rollback this change.